client/my-sites/sharing/connections/services/google-my-business.js

```diff
@@ -36,6 +38,17 @@ export class GoogleMyBusiness extends SharingService {
 		deleteStoredKeyringConnection: () => {},
 	};
 
+	externalAccessProvided = keyringConnectionId => {
+		if ( ! keyringConnectionId ) {
```

Comment:
we might want to move this to the parent class and call `super. externalAccessProvided()` here. I had some babel errors when I tried it though